### PR TITLE
IDEMPIERE-4839 IQuickForm interface

### PR DESCRIPTION
**Pull request by Soundbytes**
Jira Ticket: https://idempiere.atlassian.net/browse/IDEMPIERE-4839

### CODE REVIEW

**Review by hengsin — CHANGES_REQUESTED** [review]
First, there should be javadoc for the methods at IQuickForm interface. Second, this seems incomplete as the onQuickForm() method at AbstractADWindowContent is not touch at all.

**Comment by Soundbytes** [comment]
> First, there should be javadoc for the methods at IQuickForm interface. 
Will be added.

> Second, this seems incomplete as the onQuickForm() method at AbstractADWindowContent is not touch at all.
Great Idea! The initial idea that I have implemented here was to make the quickform dialog available as a custom form. Since the form is opened via toolbar button or menu entry I did not have to touch the onQuickForm() method.
I'll be happy to extend the patch and provide an IQuickEdit osgi service as an alternative way to access the functionality. 
I'd however prefer to add that as a separate commit.

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WQuickForm.java [hunk 1]
```diff
@@ -55,15 +55,13 @@
  * @author Logilite Technologies
  * @since Nov 03, 2017
  */
-public class WQuickForm extends Window implements EventListener <Event>, DataStatusListener
+public class WQuickForm extends Window implements IQuickForm // EventListener <Event>, DataStatusListener
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WQuickForm.java:58]: please remove the //... part

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/IQuickForm.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+	public void onIgnore();
+
+	public void onDelete();
+
+	public void onSave();
+
+	public void onRefresh();
+
+	public void dispose();
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/IQuickForm.java:44]: please add javadoc for all the methods

**Review by hengsin — CHANGES_REQUESTED** [review]
please address the requested changes.

**Review by hengsin — APPROVED** [review]
looks good.

**Pull request merged**